docs/help/changes.md

```diff
@@ -1,6 +1,38 @@
 # Changes
 
-This document outlines the changes from Version 2 (date-based version) Racks to Version 3.x Racks.
+This document outlines the changes from the older Racks to Version 3.x Racks.
+
+## Racks
+
+### Versioning
+
+- "Version 2" Racks are versioned like `20200302115619` (reverse date format, start with a 2)
+- "Version 3" Racks are versioned like `3.0.0` (Semantic versioning, start with a 3)
+
+### Underling Infrastructure
```

Comment:
Try to follow the format of the rest of this document. So for this section I'd do something like:

### Supported Infrastructure Providers

#### Version 2

* AWS (ECS)

#### Version 3

* AWS (EKS)
* Digital Ocean
* Google Cloud
* Microsoft Azure